upf/upf_ipfix.c

```diff
@@ -46,11 +46,14 @@
   do { } while (0)
 #endif
 
+/* Default report interval in seconds */
+#define UPF_IPFIX_DEFAULT_REPORT_INTERVAL 5
```

Comment:
why it's traveled from header file to here?

Reply:
It's not used outside this C file, and there's no reason to have almost whole UPG recompiled when one changes this constant